apps/explorer/src/app/routes/oracles/home/index.tsx

```diff
@@ -30,36 +38,148 @@ const Oracles = () => {
+                        .dataSourceSpecForSettlementData.status;
+                    terminationOracleStatus =
+                      o.node.tradableInstrument.instrument.product
+                        .dataSourceSpecForTradingTermination.status;
+                  } else if (
+                    o.node.tradableInstrument.instrument.product.__typename ===
+                    'Perpetual'
+                  ) {
+                    settlementOracle =
+                      o.node.tradableInstrument.instrument.product
+                        .dataSourceSpecForSettlementData.id;
+                    terminationOracle =
+                      o.node.tradableInstrument.instrument.product
+                        .dataSourceSpecForSettlementSchedule.id;
+                    settlementOracleStatus =
+                      o.node.tradableInstrument.instrument.product
+                        .dataSourceSpecForSettlementData.status;
+                    terminationOracleStatus =
+                      o.node.tradableInstrument.instrument.product
+                        .dataSourceSpecForSettlementSchedule.status;
+                  }
+                  const oracleInformationUnfiltered =
+                    data?.oracleSpecsConnection?.edges?.map((e) =>
+                      e && e.node ? e.node : undefined
+                    ) || [];
+
+                  const oracleInformation = compact(oracleInformationUnfiltered)
+                    .filter(
+                      (o) =>
+                        o.dataConnection.edges &&
+                        o.dataConnection.edges.length > 0 &&
+                        (o.dataSourceSpec.spec.id === settlementOracle ||
+                          o.dataSourceSpec.spec.id === terminationOracle)
+                    )
+                    .at(0);
+                  if (oracleInformation) {
+                    hasSeenOracleReports = true;
+                  }
 
-              const dataConnection = o?.node.dataConnection;
+                  const oracleList = `${settlementOracle} ${terminationOracle}`;
 
-              return (
-                <div
-                  id={id}
-                  key={id}
-                  className="mb-10"
-                  data-testid="oracle-details"
-                >
-                  <OracleDetails
-                    id={id}
-                    dataSource={o?.node}
-                    dataConnection={dataConnection}
-                    showBroadcasts={false}
-                  />
-                  <details>
-                    <summary className="pointer">JSON</summary>
-                    <SyntaxHighlighter data={filter(o, ['__typename'])} />
-                  </details>
-                </div>
-              );
-            })
-          : null}
+                  return (
+                    <tr
+                      id={id}
+                      key={id}
+                      className={
+                        hoveredOracle.length > 0 &&
```

Comment:
it would be much easier to do hover in css for whole row. Do we really need indicate hover only when user will mouse over OracleLink?

Reply:
I did this specifically so the rows are highlight AND the specific oracle is highlighted. I'm going to leave this in, but check with @daunatv if it's actually useful, if it isn't I'll remove this